examples/hackernews/src/components/For.astro

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
:eyes: flow components :eyes:

Not totally sold on using these in one of our example starters, happy to swap these back out for plain JSX expressions if anyone has concerns